src/swtpm/swtpm_nvstore.c

```diff
@@ -1324,10 +1324,10 @@ int SWTPM_NVRAM_PrintJson(void)
     TPM_RESULT rc = 0;
     int ret = 0, n;
     unsigned char *nvdata = NULL;
```

Comment:
`nvdata` is no longer used. We can remove this.

Reply:
Thanks. Hadn't looked. Updated.